pkgs/top-level/python-packages.nix

```diff
@@ -8380,26 +8366,18 @@ in {
 
   pytools = callPackage ../development/python-modules/pytools { };
 
-  pytorch = callPackage ../development/python-modules/pytorch {
-    cudaSupport = pkgs.config.cudaSupport or false;
-
+  pytorch = let
     # TODO: next time pytorch is updated (to 1.11.0, currently in staging as of
     # 2022-03-31), make the following changes:
-
-    # -> cudatoolk_11
-    cudatoolkit = pkgs.cudatoolkit_10;
-
-    # -> cudnn_8_3_cudatoolkit_11
-    cudnn = pkgs.cudnn_8_1_cudatoolkit_10;
-
-    # -> cutensor_cudatoolkit_11 (cutensor is a new dependency in v1.11.0)
-    # cutensor = pkgs.cutensor_cudatoolkit_11;
-
+    cudaPackages = pkgs.cudaPackages_10.overrideScope' (final: prev: {
+      # -> cudnn_8_3_cudatoolkit_11
+      cudnn = prev.cudnn_8_1_1;
+    });
     # -> setting a custom magma should be unnecessary with v1.11.0
-    magma = pkgs.magma.override { cudatoolkit = pkgs.cudatoolkit_10; };
-
-    # -> nccl_cudatoolkit_11
-    nccl = pkgs.nccl.override { cudatoolkit = pkgs.cudatoolkit_10; };
+    magma = pkgs.magma.override { inherit cudaPackages; };
```

Comment:
This looks like a no-op? Unless magma is built against a non-default `cudaPackages`?

Reply:
It's `cudaPackages` from the `let ... in` expr. But this won't be required now that staging has been merged